src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,37 @@
-import React from 'react';
+/* eslint-disable max-len */
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+interface Props {
+  selectedTodo: Todo
+  onModalClose: (value: null) => void
+}
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, onModalClose }: Props) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    const loadUsers = async () => {
```

Comment:
Generally you should assume your api calls may fail, you have no error catching so your app is prone to throwing and dying on the user.

Reply:
Also take notice that this loading and fetching logic is repeated, you could create your own custom hook that can fetch data and return { loading, value } object